.travis.yml

```diff
@@ -1,14 +1,19 @@
-language: python
+anguage: python
 python:
   - "2.7"
 env:
   - DJANGO_SETTINGS_MODULE=settings
 
 before_install:
+  - export PATH=./node_modules/.bin:$PATH
+  - sudo apt-get update -qq
+  - sudo apt-get install -y npm
```

Comment:
I'm curious if this was necessary....we have a private repo running node and we didn't need to install it. Though, the language we're using there is php instead of python.